src/lightning/pytorch/strategies/launchers/xla.py

```diff
@@ -68,8 +68,7 @@ def launch(self, function: Callable, *args: Any, trainer: Optional["pl.Trainer"]
                 a selected set of attributes get restored in the main process after processes join.
             **kwargs: Optional keyword arguments to be passed to the given function.
         """
-        context = mp.get_context(self._start_method)
-        return_queue = context.SimpleQueue()
+        return_queue = mp.Manager().Queue()
```

Comment:
The `return_queue` needs to be pickle-able to be passed throw the work queue of the multiprocessing pool that PJRT uses. The managed queue seems to work for both XRT and PJRT.

Reply:
What does "pickle-able" mean?

Reply:
PJRT `xmp.spawn()` uses `ProcessPoolExecutor` to start multi-process work. So all the args will to be put into a mp queue for the worker processes to consume. mp queue only accepts objects that can be serialized, and the mp SimpleQueue can not, so it needs to be replaced by a proxy.

Reply:
I see. It basically means `return_queue ` needs to be serializable.